omnibus/config/projects/push-jobs-client.rb

```diff
@@ -35,12 +35,14 @@
   install_dir "#{default_root}/#{name}"
 end
 
-override :bundler,        version: "1.7.12"
 # Uncomment to pin the chef version
 #override :chef,           version: "12.2.1"
```

Comment:
If you're going to use this to make a release, remember to pin chef.  You don't want to release some random untested build from master.